web_app/contract_tools/blockchain_call.py

```diff
@@ -197,7 +201,15 @@ async def get_repay_data(self, deposit_token: str, borrowing_token: str) -> dict
         pool_key["token0"], pool_key["token1"] = deposit_token, borrowing_token
         is_token1 = deposit_token == pool_key["token1"]
         supply_price = floor(await self._get_pool_price(pool_key, is_token1))
-        debt_price = floor((1 / supply_price) * 10**decimals_sum)
+
+        try:
+            debt_price = floor((1 / supply_price) * 10**decimals_sum)
+        except ZeroDivisionError:
+            logger.error(
+                f"Error while getting repay data: {deposit_token=}, {borrowing_token=}"
+            )
+            return {}
```

Comment:
add custom exeception. RepayDataException with text appropriate text about error. if there is `{}` as result, we should raise error